### DISCUSSION TITLE
Question about Error 376

### DISCUSSION THREAD

**Discussion by li2qn** [OpenRadioss/OpenRadioss#2224] [Q&A]
Hello everyone,

I am having an issue with my model. one error message saying as below. May I ask what is this error meaning? 
ERROR ID :    376
** ERROR IN USER SET (PARAMETER INDEX)
DESCRIPTION :  
   USER  PARAMETER INDEX(0),
   EXCEED (0)

Thanks
lee

**Comment by MarianBulla** [comment]
Hello li2qn,

can you provide little bit more info on what are you modelling or coding, please ?

E.g.: Are you using a user defined material model ?

Also a piece of the model, causing this error.

Thanks and best regards,
Marian

**Reply by li2qn** [reply]
Thanks for your help. I have get the issue resolved. Another issue with my model now, I am using SPH features in Radioss, I use sol2sph card to convert solid element to sph. The simulation seems able to run without error message, but in the engine out file, it keep saying that "** INFO ** SPH RE-SEARCH FOR NEIGHBOURS", do you have any idea about this message? and I have the setting as attached. Thanks for any suggestions.
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/5f5f6d3a-ab96-4e1f-a8e5-40d0fe30c931)

**Reply by li2qn** [reply]
Adding to the above questions, I have noticed some particles penetrate through the rigid wall which not supposed to. Please see the attached image. I also attached the image for interface setting. any idea of what is going on here? Only gravity load add on the particles. please help with any suggestions. Thanks
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/5261f6b8-325c-48b9-9132-1aab8407c8ec)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/2ed5417f-66f3-4024-a5dd-f4afb03732de)
![image](https://github.com/OpenRadioss/OpenRadioss/assets/48695377/5a515fd4-792f-4f5b-93a1-4cdf8baa523a)

**Comment by li2qn** [comment]
Need help here. Thanks for all suggestions and comments
